tools/mo/openvino/tools/mo/utils/versions_checker.py

```diff
@@ -231,6 +235,73 @@ def get_environment_setup(framework):
     return env_setup
 
 
```

Comment:
we also need this parsing functionality for requirements and constrain files in `setup.py`. There is already implemented it there but not quire much well.
Please create the common utils routines for both `setup.py` and MO and use it.

Reply:
Removed checks of unsatisfied dependencies as discussed offline.